client/stylesheets/_toCat.scss

```diff
@@ -4346,12 +4349,55 @@ On click the images will be displayed at normal size to complete the effect
     margin-left: 5px;
     vertical-align: middle;
     color: #fff !important;
+} */
+
+.sortClicker i {
+	margin-left: 8px;
+	margin-bottom: -2px;
+	color: #fff !important;
+}
+
+.sortClick {
+	float: right !important;
+    margin-right: 30px;
+    font-size: 0.8rem;
+}
+
+.sortItems {
+	margin-top: -2px;
+}
+
+.sortItems li {
+	width: 100%;
+}
+
+.sortClick ul {position:relative;font-weight:400;}
+.sortClick ul li {position:relative;list-style:none;cursor:pointer;}
+.sortClick ul li ul {position:absolute;left:0;right:0;border: 2px solid #04b2ec;}
+.sortClick ul .sortClicker {position:relative;background:#00bfff;color:#FFF;}
+.sortClick ul .sortClicker:hover,.click-nav ul .active {background:#04B2EC !important;}
+.sortClick img {position:absolute;top:9px;left:12px;}
+.sortClick ul li a {
+	display:block;padding:10px 10px;background:#FFF;color:#333;text-decoration:none;
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`<br>Line should be indented with spaces, not tabs<br>Properties should be ordered background, color, display, padding, text-decoration<br>Shorthand form for property `padding` should be written more concisely as `10px` instead of `10px 10px`<br>Property 'padding' should be placed on own line<br>Property 'background' should be placed on own line<br>Property 'color' should be placed on own line<br>Property 'text-decoration' should be placed on own line<br>Colon after property should be followed by one space